src/main/webapp/data/userMapData.json

```diff
@@ -726,7 +748,8 @@
       "University of Auckland",
       "University of Canterbury",
       "University of Otago",
-      "University of Waikato"
+      "University of Waikato",
```

Comment:
unnecessary

Reply:
Hi @shaharyar-shamshi  , As you can see comma was not there so i added it.